scripts/docker_files/docker_file_wheelbuilder_linux_mpi/Dockerfile

```diff
@@ -34,6 +34,8 @@ RUN git clone -b 'v5.5.1' --depth 1 https://github.com/MmgTools/mmg /workspace/m
     rm -r /workspace/mmg_5_5_1 && \
     cd /
 
+RUN python3.6 -m pip install mypy
```

Comment:
that's it, but you need to do it for 3.6, 3.7, 3.8, 3.9 and 3.10

I think the binary are located at `/opt/python/VERSION/bin/python`

Reply:
3.6 is already legacy...

Reply:
I am doing it in the following place also. 
https://github.com/KratosMultiphysics/Kratos/blob/950b67f72ebe36c1127f9f4c52814aaf1d760da3/scripts/wheels/linux/build.sh#L161

I saw the core is built against pytho3.6, and thats why i included it in there as well. Am i wrong?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/950b67f72ebe36c1127f9f4c52814aaf1d760da3/scripts/wheels/linux/build.sh#L149

Reply:
No you both are right. Sadly 3.6 is still not legacy for us (but it soon will be if it depends on me :imp: ).

The build process in the wheel builder is different from the one you are used in regular builds. In order to save time we make a single compilation of the "Core" .so and we use those to link agains different version of the "application" ,so. The core elements do not depend on python, but I didn't want to rewrite all the cmake files so I simply pass a placeholder python version so it does not complain.

So we build:
KratosCore.so

Then
Kratos.p36.so, Kratos.p37.so ... Kratos.p310.so

Which all link to the same Kratos.so. But the commands in the cmake builf will be invoked with the proper version of python (the one inside the loop below the build_core instruction)

Reply:
In that case we don't need to have the following ryt?
`RUN python3.6 -m pip install mypy`, because it is anyways installed for the respective version from `$PYTHON_LOCATION -m pip install mypy ` ryt?

Reply:
aaa you are right sorry, didn't see that change! Exactly as you say.

Reply:
@roigcarlo  done :)